addons/website/static/src/components/configurator/configurator.xml

```diff
@@ -54,9 +54,9 @@
                     </div>
                     <span t-att-class="!state.selectedType ? 'o_configurator_hide' : 'o_configurator_show'"> for my</span>
                 </div>
-                <div t-attf-class="o_configurator_typing_text d-inline d-md-block o_configurator_industry mb-md-2 mb-lg-4 {{!state.selectedType ? 'o_configurator_hide' : 'o_configurator_show'}}">
-                    <div class="o_configurator_industry_wrapper position-relative d-inline">
-                        <i t-attf-class="industry_selection d-inline d-md-inline-block rounded bg-100 px-2 px-md-3 mx-2 mx-md-0 {{state.selectedIndustry ? 'o_step_completed' : 'o_step_todo show'}}" contenteditable="True" t-on-blur="blurIndustrySelection" t-on-input="inputIndustrySelection" t-ref="industrySelection"/>
+                <div t-attf-class="o_configurator_typing_text d-inline d-md-flex align-items-start o_configurator_industry mb-md-2 mb-lg-4 {{!state.selectedType ? 'o_configurator_hide' : 'o_configurator_show'}}">
+                    <div class="o_configurator_industry_wrapper position-relative d-inline mr-2">
+                        <i t-attf-class="industry_selection overflow-hidden d-inline d-md-inline-block rounded bg-100 px-2 px-md-3 mx-2 mx-md-0 {{state.selectedIndustry ? 'o_step_completed' : 'o_step_todo show'}}" contenteditable="True" t-on-blur="blurIndustrySelection" t-ref="industrySelection"/>
```

Comment:
Oh my god... are we really using `<i/>` elements instead of `<input/>` ones? :disappointed: Why? :cry: I cannot think of any reason to want this. The simple fact I see the use of the "overflow-hidden" class for a design where nothing overflows anything tells me there something wrong going on here :/

Then we also have issues like this one:
![Screenshot from 2021-04-28 10-52-27](https://user-images.githubusercontent.com/10338094/116376015-1fb79480-a810-11eb-97bb-709318614e4a.png)

Reply:
@stefanorigano I remember there was a reason why we used `<i/>` instead of `<input/>` but I can't remember it...

Reply:
@qsm-odoo 

> Oh my god... are we really using <i/> elements instead of <input/> ones? 😞 Why? 

The field length should fit its content dynamically. I used CSS, but JS is an option too... I'm just not comfortable coding it (too much events to handle etc..).

`overflow-hidden` has been added to fix the issue you have spot of Firefox. Apparently it can't correctly compute the field's rendering dimensions (because of `contenteditable="True"` + flexbox ?).
I found it very difficult to debug, nothing on the CSS side ringed a bell: line-height, flex proprieties..  all seems fine.
I suspect a Firefox rendering issue, indeed Chrome and Safari are fine.
Forcing a fixed height + overflow solved the issue.

Reply:
`overflow-hidden` does not fix issues, it hides them :wink: I think that simplifying the DOM and using input elements will solve all of these problems without hacks or things we don't understand or don't even see on our current browsers and screens.

However the auto-resize of the input is indeed a good reason to be afraid to use them... but I would have been more afraid to use `contenteditable` as a solution, you don't know what you lose in accessibility / keyboard navigation etc. I found a simple solution to autoresize inputs with one line JS and really not much CSS: https://jsfiddle.net/we6um8fj/

Reply:
For the second issue using `min-width: 100%` instead of `width: 100%` on the dropdown menu give the following result. Does it look good to you @qsm-odoo or should the selected item be always as large as the dropdown menu ? (which is not the case here for 'a blog'). 
![Screenshot from 2021-04-28 12-23-50](https://user-images.githubusercontent.com/49784121/116389505-4d570a80-a81d-11eb-8e1b-16cc18c7bdcb.png)
![Screenshot from 2021-04-28 12-23-44](https://user-images.githubusercontent.com/49784121/116389508-4defa100-a81d-11eb-8bc6-6551498773a3.png)

Reply:
Well, yes it is obviously good... :)

Reply:
> Well, yes it is obviously good... :)

Ok I'll include that and a fix for `<i/>` to `<input/>` in this commit then